buku

```diff
@@ -534,13 +550,11 @@ class BukuDb:
 
         Returns
         -------
-        tuple or None
+        BookmarkVar or None
```

Comment:
Wouldn't this affect third-party utilities? Would their calls work seamlessly?

Reply:
If you're talking about replacing a tuple with a named-tuple, then it's not an issue – named-tuple is just a regular tuple with a few extra attributes that give access to the same values as the positional indices (thus providing a more convenient usage interface _in addition_ to the existing one). It even passes `isinstance` check for the `tuple` type.

Reply:
:+1: